packages/paypal-commerce-integration/src/paypal-commerce-ratepay/paypal-commerce-ratepay-initialize-options.ts

```diff
@@ -0,0 +1,42 @@
+import { BirthDate } from '../paypal-commerce-types';
+
+export interface PaypalCommerceRatePay {
+    /**
+     * The CSS selector of a container where the payment widget should be inserted into.
+     */
+    container: string;
+
+    /**
+     * The CSS selector of a container where the legal text should be inserted into.
+     */
+    legalTextContainer: string;
+
+    /**
+     * Callback that triggers pending logic (for example: show loading overlay)
+     */
+    onPaymentSubmission?(isPaymentSubmitting: boolean): void;
+
+    /**
+     * A callback that gets form values
+     */
+    getFieldsValues?(): {
+        ratepayBirthDate: BirthDate;
+        ratepayPhoneNumber: string;
+        ratepayPhoneCountryCode: string;
+    };
+
+    /**
+     * A callback right before render Smart Payment Button that gets called when
+     * Smart Payment Button is eligible. This callback can be used to hide the standard submit button.
+     */
+    onRenderButton?(): void;
+
+    /**
+     * A callback for displaying error popup. This callback requires error object as parameter.
+     */
+    onError?(error: unknown): void;
+}
+
+export interface WithPayPalCommerceRatepay {
```

Comment:
This interface should be updated to
`WithPayPalCommerceRatePayPaymentInitializeOptions`, because this method will not be used to generate the interfaces for Resolve strategies, therefore there might be a build error.

Reply:
renamed